docs/schema.md

```diff
@@ -667,6 +678,7 @@ The defined ecosystems are:
 | `Photon OS` | The Photon OS package ecosystem; the `name` is the name of the RPM package. The ecosystem string must have a `:<RELEASE-NUMBER>` suffix to scope the package to a particular Photon OS release. Eg `Photon OS:3.0`. |
 | `Pub` | The package manager for the Dart ecosystem; the `name` field is a Dart package name. |
 | `PyPI` | the Python PyPI ecosystem; the `name` field is a [normalized](https://www.python.org/dev/peps/pep-0503/#normalized-names) PyPI package name.  |
++| `Red Hat` | The Red Hat package ecosystem; the `name` field is the name of a binary RPM. The ecosystem string is a translation of Red Hat [Common Platform Enumerations](https://cpe.mitre.org/) (CPEs). The suffix of Red Hat CPEs are used as ecosystem idenfiers with the `cpe/:[oa]:(redhat)` prefixes replaced with `Red Hat`. Red Hat ecosystem identifiers can be used to idenfity vulnerability RPMs installed on a Red Hat system as explained [here](https://www.redhat.com/en/blog/how-accurately-match-oval-security-data-installed-rpms).  |
```

Comment:
Fix spelling and grammar
```suggestion
+| `Red Hat` | The Red Hat package ecosystem; the `name` field is the name of a binary RPM. The ecosystem string is a translation of Red Hat [Common Platform Enumerations](https://cpe.mitre.org/) (CPEs). The suffix of Red Hat CPEs are used as ecosystem identifiers with the `cpe/:[oa]:(redhat)` prefixes replaced with `Red Hat`. Red Hat ecosystem identifiers can be used to identify vulnerable RPMs installed on a Red Hat system as explained [here](https://www.redhat.com/en/blog/how-accurately-match-oval-security-data-installed-rpms).  |
```

Reply:
Is the "binary RPM" part accurate? I thought I saw source RPMs getting mentioned in the conversion as well?